.eslintrc.json

```diff
@@ -31,6 +36,9 @@
         "html": "ignore",
         "custom": "ignore"
       }
-    ]
+    ],
+    "linebreak-style": 0,
```

Comment:
You can remove this rule. Do `ctrl + shift + P` and search for user settings. In User settings, search `eol` and change it `\n`.
Also, search for `eslint: auto fix on save` and make sure it is checked. It will do what the name suggests. Now, `ctrl + s` the files that are giving you errors.

Reply:
Modified...